unit2.js

```diff
@@ -0,0 +1,68 @@
+document.addEventListener("DOMContentLoaded", () => {
+    let select = document.querySelector("select")
+    let form = document.querySelector("form")
+    let ul = document.querySelector("ul")
+
+    const getMovies = async () => {
+        try {
+            let res = await axios.get("https://ghibliapi.herokuapp.com/films")
+            let films = res.data;
+            // debugger
+            films.forEach(film => {
+                let option = document.createElement("option")
+                let selected = document.querySelector("select")
```

Comment:
Since you're already querySelected outside this scope, you don't have to define selected here again, you can just use your `select` variable. Doing this here might be the cause of your select disappearing after the first query.